docs/ref/contrib/admin/index.txt

```diff
@@ -2861,6 +2861,43 @@ Templates can override or extend base admin templates as described in
 ``AdminSite`` methods
 ---------------------
 
+.. method:: AdminSite.get_app_list(request, app_label=None)
+
+    .. versionadded:: 4.1
```

Comment:
This method exists for a long time, so I would remove `versionadded` annotation and only add `versionchanged` about `app_label` argument:
```
.. method:: AdminSite.get_app_list(request, app_label=None)

    ...

    .. versionchanged:: 4.1

        The ``app_label`` argument was added.

.. method:: AdminSite.has_permission(request) 
```